core-skills/c-sharp-and-dotnet/README.md

```diff
@@ -0,0 +1,78 @@
+# C# and .NET Core
+
+## Who is this skill for?
+
+This skill is intended for developers who have already completed the TDD core skill, and who have not worked with .NET previously (or want to brush up) and are likely to join a delivery team that is working in C#.
+Through this course, we'll be making some assumptions that you're already familiar with core Made Tech processes and at least one programming language (static typed or dynamic).
```

Comment:
"Made Tech processes" ?

Reply:
"the Made Tech way"?

Reply:
why does this matter? can't see anythign that references it